src/adapters/adapter.js

```diff
@@ -66,7 +65,7 @@ class Adapter {
             // Uses `type` as class name for tree node
             item.icon = type;
 
-            octotree.setNodeIconAndText(this, item);
+            await octotree.setNodeIconAndText(this, item);
```

Comment:
Is this async fn?

Reply:
Yes, there is a store.get inside
https://github.com/ovity/octotree/blob/master/src/core.api.js#L81